controllers/users.js

```diff
@@ -602,6 +604,51 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Collects all Users with ONBOARDING state and are present in discord server for more than 31 days
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getUsersWithOnboardingState = async (req, res) => {
+  const minimumPresentDaysInServer = 31;
+  try {
+    const { allUserStatus } = await userStatusModel.getAllUserStatus(req.query);
+
+    const allUsersWithOnboardingState = filterUsersWithOnboardingState(allUserStatus);
```

Comment:
Would be great if we fetch only users who are in onboarding state instead of fetching all users & filtering them in a middleware IMO